overkill.rb

```diff
@@ -17,7 +17,7 @@ def run
 
     def apps_to_kill
       apps = {
-        "iTunes" => "🎵"
+        "Music" => "🎵"
```

Comment:
Ohh nice, I'm still on the old macOS, let's keep both `iTunes` and `Music`, looks like I can't update the PR, mind doing that quick change? Thanks